MCprep_addon/mcprep_ui.py

```diff
@@ -609,6 +627,28 @@ def draw(self, context):
 			col = split.column()
 			p = col.operator("mcprep.openfolder", text=env._("Open texture pack folder"))
 			p.folder = self.custom_texturepack_path
+			
+			row = layout.row()
+			row.scale_y = 0.7
+			row.label(text=env._("Refresh MCprep data with local Minecraft install"))
+			box = layout.box()
+			split = util.layout_split(box, factor=factor_width)
+			col = split.column()
+			col.label(text=env._("Path to Minecraft versions folder"))
+			col = split.column()
+			col.prop(self, "save_folder", text="")
+			split = util.layout_split(box, factor=factor_width)
+			col = split.column()
+			col.label(text=env._("Refresh"))
+			col = split.column()
+			col.operator("mcprep.refresh_data", text=env._("Locally Refresh MCprep JSON data"))
+			col.enabled = False
+			
+			if not os.path.isdir(bpy.path.abspath(self.save_folder)):
+				row = box.row()
+				row.label(text=env._("Saves folder not found"), icon="ERROR")
```

Comment:
Reads a bit more consistently with above changes.
```suggestion
				row.label(text=env._("Versions folder not found"), icon="ERROR")
```